CHANGELOG.md

```diff
@@ -4,7 +4,23 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [0.5.2] - 2021-MM-DD
+## [0.5.3] - 2021-05-29
+
+### Added
+- Added inverse for augmentations ([#1013](https://github.com/kornia/kornia/pull/1013))
+- Add more advenced augmentations ([#1015](https://github.com/kornia/kornia/pull/1015)
+
+### Fixed
+- Correct Sobel test_noncontiguous. Nothing was tested before. ([#1018](https://github.com/kornia/kornia/pull/1018))
+- Fixing #795: find_homography_dlt_iterated sometimes fails ([#1022](https://github.com/kornia/kornia/pull/1022))
+
+### Changed
+- PoC: build multi-OS Python ([#1017](https://github.com/kornia/kornia/pull/1017)) 
+- Refactorization of the morphology package ([#1034](https://github.com/kornia/kornia/pull/1034))
```

Comment:
Add that it is a breaking change, as the function behaviour is changed - before it was not correct

Reply:
Morphology, I mean

Reply:
in theory breaking changes are included in this category. We are following this convention: https://keepachangelog.com/en/1.0.0/